corehq/apps/cloudcare/static/cloudcare/js/formplayer/js/apps/menus/menu_view.js

```diff
@@ -0,0 +1,117 @@
+/*global FormplayerFrontend */
+
+FormplayerFrontend.module("SessionNavigate.MenuList", function (MenuList, FormplayerFrontend, Backbone, Marionette) {
+    MenuList.MenuView = Marionette.ItemView.extend({
+        tagName: "tr",
+        template: "#menu-view-item",
```

Comment:
if this is a script template, probably makes sense to suffix with `-template` like you did with the others